data/lua/carryover_gold.lua

```diff
@@ -19,17 +22,19 @@ local function get_num_persistent_teams()
 	return res
 end
 
-local function get_num_villages()
+--- Gets the number of villages on the map.
+function carryover.get_num_villages()
 	return #(wesnoth.map.find{ gives_income = true })
 end
 
-local function half_signed_value(num)
+--- Like tostring(num) but uses a prettier minus sign, to be used in the UI.
```

Comment:
This is clearly not true, it's just returning `tostring(num)` directly.

Also, this seems like something that probably belongs in either the `mathx` or `stringx` module, not in here.

Reply:
Tbh i don't even know why it's called that i just copied the name from the c++ function, but yeah the implementation is not there yet, idk whether it'd be really useful another places or whether other places usually want to use the normal minus sign.

Reply:
If it's a C++ function, we could just make this a thin wrapper around the C++ function. We should at least consider changing the name though, as it doesn't make sense to me.

I also think it should return a `t_string`, rather than a regular Lua string.

Reply:
This still applies now.